helm/crossplane/values.yaml

```diff
@@ -1,10 +1,108 @@
-image:
-  repository: crossplane/crossplane
-  tag: v1.7.0
-  pullPolicy: IfNotPresent
+giantswarm:
+  images:
+    registry: quay.io
+    crossplane:
+      image: giantswarm/crossplane
+    pullPolicy: IfNotPresent
+
+  crds:
+    install: true
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+
+replicas: 1
+
+deploymentStrategy: RollingUpdate
+
+nodeSelector: {}
+tolerations: {}
+affinity: {}
+
+# -- Custom labels to add into metadata
+customLabels: {}
+
+# -- Custom annotations to add to the Crossplane deployment and pod
+customAnnotations: {}
+
+# -- Custom annotations to add to the serviceaccount of Crossplane
+serviceAccount:
+  customAnnotations: {}
+
+leaderElection: true
+args: {}
 
 provider:
   packages: []
 
 configuration:
   packages: []
+
+imagePullSecrets:
+- dockerhub
```

Comment:
Where can I find this secret?

Reply:
It came from upstream. By chance, the very latest commit at the time in upstream removes it: https://github.com/crossplane/crossplane/commit/43c577253f6babcf5002944fa7e1d5e55cc74f45

Reply:
But latest release it is like this: https://github.com/crossplane/crossplane/blob/6b674f868403e92fa40d49a8a0385352d8d32e39/cluster/charts/crossplane/values.yaml.tmpl#L33-L34